project-koku/koku
# fix OCI serializer order-by

## Description

This change will fix the serializer for OCI order-bys.

## Testing

[http://localhost:8000/api/cost-management/v1/reports/oci/costs/?delta=cost&filter[limit]=10&filter[offset]=0&filter[resolution]=monthly&filter[time_scope_units]=month&filter[time_scope_value]=-1&group_by[payer_tenant_id]=*&order_by[payer_tenant_id]=asc&currency=EUR](http://localhost:8000/api/cost-management/v1/reports/oci/costs/?delta=cost&filter[limit]=10&filter[offset]=0&filter[resolution]=monthly&filter[time_scope_units]=month&filter[time_scope_value]=-1&group_by[payer_tenant_id]=*&order_by[payer_tenant_id]=asc&currency=EUR)

1. On main, the above query results in a 500 Internal Error
2. Checkout branch
3. See results return as 200 status.

## Notes

...
